### DISCUSSION TITLE
when creating new db

### DISCUSSION THREAD

**Discussion by bkyerv** [planetscale/discussion#359] [Q&A] [answered]
When new app is build and new db is set up, how does one create all tables? Should the queries be triggered by hitting a certain api? or is it better to manually run some script that gets run once only at the very beginning? if so, what is the best place to put such script? how does one trigger execution of such script once only?

**Comment by fooforge** [comment, marked as answer]
Hey @bkyerv,

PlanetScale is no different from any other MySQL database in that sense. You can connect directly to your database and issue SQL queries with your preferred `mysql` client or from the _Shell_ tab in our web-interface. When you use a framework such as [Prisma](https://www.prisma.io/) to develop your application, chances are, the framework will generate your database schema and necessary migrations for you. In that case, you would normally interact with your database by issuing commands such as `npx prisma db push` or similar.

For more information on how to get started with PlanetScale, check out [our official documentation](https://planetscale.com/docs) (especially [this quick start guide](https://planetscale.com/docs/tutorials/planetscale-quick-start-guide)), and check out the [tutorial section in our blog](https://planetscale.com/blog/category/tutorials).

**Reply by bkyerv** [reply]
@fooforge  thank you, Mike
